conclave-host/src/main/kotlin/com/r3/conclave/host/internal/gramine/GramineEnclaveHandle.kt

```diff
@@ -54,6 +63,16 @@ class GramineEnclaveHandle(
         enclaveInterface = SocketHostEnclaveInterface()
     }
 
+    private fun runSimpleCommand(vararg command: String): String {
+        val process = ProcessBuilder()
+            .command(command.asList())
+            .start()
+        process.waitFor(5L, TimeUnit.SECONDS)
+        return process.inputStream.bufferedReader().use(BufferedReader::readText).trimEnd()
+            .removeSuffix(System.lineSeparator())
```

Comment:
There's some redundancy here and over-complication. Just do `process.inputStream.reader().use { it.readText() }.trimEnd()`